### PR TITLE
Copy my to-do-list-app code

### PR DESCRIPTION
**Pull request by lily-coder**

### CODE REVIEW

File: src/clear.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { setStorage } from './storage.js';
+
+export default (arr) => {
+  const toremove = [];
+  for (let i = 0; i < arr.length; i += 1) {
+    if (arr[i].completed == 'true') { toremove.push(i); } // eslint-disable-line
```
Comment (brytebee) [inline src/clear.js:6]: Use strong type check
Instead of `==` use `===`

**Review by brytebee — COMMENTED** [review]
Suggested Changes ♻️ 

Hi, @lily-coder!
You have done an amazing job!
Well done!

File: src/clear.js (hunk 1 above)
Comment (lily-coder) [reply src/clear.js]: I have made the changes as requested. Thank you for the review

**Review by brytebee — APPROVED** [review]
# Approved 👏👏

@lily-coder!

Congratulations! You have done an outstanding job👍.
There's nothing more to say, than it's time to merge👊 this branch and complete it.

🥂🥂

**Comment by lily-coder** [comment]
Thank you

**Pull request merged**